pytorch_lightning/accelerators/ddp_spawn_accelerator.py

```diff
@@ -167,8 +167,6 @@ def ddp_train(self, process_idx, mp_queue, model, is_master=False, proc_offset=0
         # 16-bit
         model = self.trainer.precision_connector.connect(model)
 
-        self.trainer.convert_to_lightning_optimizers()
```

Comment:
not related now, but as it is in each then it shall be in the base not in all children...

Reply:
I couldn't do as the self.trainer.precision wasn't called as the same place depending of the accelerator.
It would have required a cleaning of accelerators.